workflows/data_pipelines/rne/database/process_rne.py

```diff
@@ -140,17 +140,30 @@ def inject_records_into_db(file_path, db_path, file_type):
                         data, file_type
                     )
                     unites_legales += unites_legales_temp
+
+                    if len(unites_legales) > 100000:
+                        insert_unites_legales_into_db(
+                            unites_legales, file_path, db_path
+                        )
+                        unites_legales = []
+
```

Comment:
I dont get this part. What happens if `len(unites_legales) < 100000` ?

Reply:
I added this to deal with cases where typically flux file is large. Before, all the companies in one flux files will be added to `unites_legales` list, and once the loop is finished the list will be injected into the database.
However, is if the list gets too large, momory runs out.
The `>100000` forces list to be injected into database in chucks of 100000 and reset `unites_legales` list, instead of waiting for entire flux file to be processed and inserted in one shot.

Reply:
I guess a two line comment is welcome here. 

"When pending queue reach 100 000 we insert it directly otherwise it will be inserted at the end of the loop"